dotcom-rendering/src/lib/identity.ts

```diff
@@ -26,22 +33,35 @@ const getRedirectUri = (stage: StageType) => {
 	}
 };
 
-let identityAuth: IdentityAuth | undefined = undefined;
+let identityAuth: IdentityAuth<never, CustomIdTokenClaims> | undefined =
+	undefined;
 
 function getIdentityAuth() {
 	if (identityAuth === undefined) {
 		const stage = getStage();
 
-		identityAuth = new IdentityAuth({
+		identityAuth = new IdentityAuth<never, CustomIdTokenClaims>({
 			issuer: getIssuer(stage),
 			clientId: getClientId(stage),
 			redirectUri: getRedirectUri(stage),
-			scopes: ['openid', 'profile', 'email'], // and any other scopes you need
+			scopes: [
+				'openid', // required for open id connect, returns an id token
+				'profile', // populates the id token with basic profile information
+				'email', // populates the id token with the user's email address
+				'guardian.discussion-api.private-profile.read.self', // allows the access token to be used to make requests to the discussion api to read the user's profile
+				'guardian.discussion-api.update.secure', // allows the access token to be used to make requests to the discussion api to post comments, upvote etc
+				'guardian.identity-api.newsletters.read.self', // allows the access token to be used to make requests to the identity api to read the user's newsletter subscriptions
+				'guardian.identity-api.newsletters.update.self', // allows the access token to be used to make requests to the identity api to update the user's newsletter subscriptions
+				'guardian.members-data-api.read.self', // allows the access token to be used to make requests to the members data api to read the user's membership status
+				'id_token.profile.theguardian', // populates the id token with application specific profile information
+			],
 		});
 	}
 	return identityAuth;
 }
 
-export async function isSignedInWithOkta(): Promise<boolean> {
-	return getIdentityAuth().isSignedIn();
+export async function isSignInWithOktaAuthState(): Promise<
```

Comment:
Very nitpicky, but guess this should be: 

```suggestion
export async function isSignedInWithOktaAuthState(): Promise<
```